themes/stanford_basic/decoupled-menu/src/main-menu.island.tsx

```diff
@@ -198,16 +199,30 @@ const Button = styled.button`
 
   &:hover, &:focus {
     box-shadow: none;
-    border-bottom: 2px solid #ffffff;
+    border-bottom: 1px solid #ffffff;
     background: transparent;
   }
 
+  @media (max-width: 992px) {
+    height: 38px;
+    background-color: #b1040e;
+    background-size: contain;
+    width: 38px;
+    border-radius: 38px;
+
+    &:hover, &:focus {
+      box-shadow: none;
+      border-bottom: none;
+      background: #b1040e;
+    }
+  }
```

Comment:
this produces a odd behavior at exactly 992px wide screen. if you use max-width, it should be 991. It would be better to just add the styles outside of the media query, then adjust them in the min-width query directly below this portion.

And the hover/focus is just overridding styles directly above. These are needed for a11y. There has to be a visual identifier such as a border/underline/etc that indicates activity. It doesn't match the current menu because the current menu is not done correctly.

Reply:
I wanted to replicate the red circle with the rotation. But you are right about two visual changes. So, I took this part out.